### PR TITLE
#16721, #16227, #15770: Initial changes to my-business registry dashb…

### PR DESCRIPTION
**Pull request by marzmehr**
*Issue #:*
https://github.com/bcgov/entity/issues/16721
https://github.com/bcgov/entity/issues/16227
https://github.com/bcgov/entity/issues/15770

*Description of changes:*
Initial UI changes
Note: API integration has been completed for business search.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Why was this merged before review?

**Comment by marzmehr** [comment]
Because it's not ready to be merged into the main branch yet.

**Comment by severinbeauvais** [comment]
OK. But it's nicer to review several small PRs instead of one large one.

**Comment by severinbeauvais** [comment]
@pwei1018 @seeker25 Could we get gcbrun to work in this repo/branch, please?

File: auth-web/src/components/auth/manage-business/BusinessLookup.vue [hunk 1]
```diff
@@ -124,7 +128,7 @@ export default class BusinessLookup extends Vue {
     // safety check
     if (input) {
       // change to summary state
-      this.state = States.SUMMARY
+      // this.state = States.SUMMARY
     }
     // event data is automatically returned
   }
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/BusinessLookup.vue:134]: If no body is needed for this emitter method then please simplify it, ie,
```
@Emit('business')
  onItemSelected (input: BusinessLookupResultIF): void {}
```

**Comment by seeker25** [comment]
Patrick question - I'll deploy this to DEV for now

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -522,6 +538,12 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     this.$refs.errorDialog.open()
   }
 
+  showBusinessAlreadyAdded (event) {
+    this.dialogTitle = 'Business Already Added'
+    this.dialogText = `The business ${event.name} with the businss number ${event.identifier} is already in your Business Registry List.`
+    this.$refs.errorDialog.open()
+  }
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:545]: Why is this needed, since already-existing businesses should appear as "Added" in the search results and not be clickable?

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 3]
```diff
@@ -737,10 +759,17 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
...
   float: right;
   width: 200px;
+  height: 10% !important;
+  transform: translate(-10px,42px);
+  z-index: 1;
 }
 
 // Vuetify Overrides
 ::v-deep {
+  .column-selector.v-text-field .v-input__control {
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:769]: Could you add a comment indicating what/why you need this override? Thanks.

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+                      Confirm authorization using your registered office email address
+                      <div class="subtitle"> (If you forgot or don't have a business {{passwordText}})</div>
+                    </v-list-item-title>
+                  </template>
+                  <div style="color:#313132;">
+                    <div>
+                      An email will be sent to the registered office contact email of the business:
+                    </div>
+                    <div><b>{}**@********</b></div>
+                    <div style="margin:6px 5px 16px 0 !important">
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:93]: I see a lot of hard-coded styles in this template, which should use classnames instead.

Also it looks like line 92 is for debugging -- delete now or remember to delete it later :)

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+    &.top-of-list{
+      border-top: 1px solid rgb(228, 228, 228);
+    }
+    .item-content{
+      color: #000 !important;
+    }
+  }
+
+  .v-list-item{
+    background: #FFFFFF;
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:420]: I'd prefer to see `white` or `black` here and elsewhere -- no hex colours.

If you need a new colour then add a colour variable to theme.scss.

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 6]
```diff
@@ -0,0 +1,506 @@
...
+    height: 4rem !important;
+    margin: 0 !important;
+  }
+
+  .v-list-item--link>
+  .v-list-item__title{
+    font-weight: 300 !important;
+    margin-left:-1rem !important;
+    color: var(--v-primary-base) !important;
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:429]: This is a way to get the Vuetify colour, I think, but there is probably an existing colour variable you can use more simply here (something like `color: $app-blue`).

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 7]
```diff
@@ -0,0 +1,506 @@
...
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '$assets/scss/theme.scss';
+
+::v-deep {
+
+  .v-list-group{
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:409]: It would be nice to have comments on the styling overrides, so the next developer working in this code can more easily understand why they're here.

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 8]
```diff
@@ -0,0 +1,506 @@
...
+//     opacity: 0.4;
+//   }
+// }
+
+// // remove whitespace below error message
+// .authorization {
+//   ::v-deep .v-text-field__details {
+//     margin-bottom: 0 !important;
+//   }
+// }
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:505]: Before the final commit/merge in this branch, obsolete code needs to be deleted, please.

**Comment by marzmehr** [comment]
This PR has been transferred to https://github.com/bcgov/sbc-auth/pull/2376 to follow correct feature branch naming conventions.
